hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -199,6 +194,22 @@ module top_${top["name"]} #(
   end
 % endif
 
+  % if len(top["inter_signal"]["definitions"]) >= 1:
+  // define inter-module signals
+  % endif
+  % for sig in top["inter_signal"]["definitions"]:
+    % if sig["type"] == "req_rsp":
+  ${sig["package"]}::${sig["struct"]}_req_t ${lib.bitarray(sig["width"],1)} ${sig["signame"]}_req;
+  ${sig["package"]}::${sig["struct"]}_rsp_t ${lib.bitarray(sig["width"],1)} ${sig["signame"]}_rsp;
+    % elif sig["type"] == "broadcast":
+      % if sig["struct"] == "logic":
+  logic ${lib.bitarray(sig["width"],1)} ${sig["signame"]};
+      % else:
+  ${sig["package"]}::${sig["struct"]}_t ${sig["signame"]};
+      % endif
+    % endif
+  % endfor
+
```

Comment:
The previous location was probably dumbest thing I ever done...Why it was there inside eflash? Sorry for that.. haha

Reply:
dont be too harsh on yourself :) it was the only module up to this point that had inter-module, it is totally understandable!